src/helpers/commentHelpers.js

```diff
@@ -0,0 +1,27 @@
+import responseGenerator from './responseGenerator';
+
+const commentHelpers = {
+  liker: (model, res, likeData, type) => {
+    model
+      .findOrCreate({ where: likeData, defaults: { liked: true } })
+      .spread((result, created) => {
+        const { liked } = result.dataValues;
+        // When user is liking for the first time
+        if (created) {
+          return responseGenerator.sendSuccess(res, 201, `${type} liked`);
+        }
+
+        // This block creates the toggle effect of liking and unliking
+        return model.update({ liked: !liked }, { where: likeData }).then(() => {
```

Comment:
Hello, this should be implemented as `remove` since we would not be having a dislike feature on the project.
The first request to this api adds the like for such comment while the second removes the likes